### PR TITLE
Add IPython notebook with feedback analysis

### PR DESCRIPTION
**Pull request by encukou**

### CODE REVIEW

File: stats/B161/feedback-analysis.ipynb [hunk 1]
```diff
@@ -0,0 +1,1130 @@
...
+   "metadata": {},
+   "source": [
+    "# MI-PYT feedback"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Feedback for the [MI-PYT course](https://github.com/cvut/MI-PYT) was requested through a [feedback form application](https://github.com/encukou/feedback-form). The results are [public](results.csv).\n",
```
Comment (hroncok) [inline stats/B161/feedback-analysis.ipynb:14]: not sure if the relative link works on GitHub render, it appears to not work when I look at your repo

File: stats/B161/feedback-analysis.ipynb [hunk 2]
```diff
@@ -0,0 +1,1130 @@
...
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>5.0</th>\n",
+       "      <td>10.0</td>\n",
+       "
```
Comment (hroncok) [inline stats/B161/feedback-analysis.ipynb]: > but I'm struggling to find a good method to test this

Maybe plot a histogram of average rating grouped by students?

**Review by hroncok — APPROVED** [review]
I like it. The comments are nitpicks.

File: stats/B161/feedback-analysis.ipynb (hunk 1 above)
Comment (encukou) [reply stats/B161/feedback-analysis.ipynb:14]: Good catch. I'll add an absolute URL in another commit, so I can copy it.

File: stats/B161/feedback-analysis.ipynb (hunk 2 above)
Comment (encukou) [reply stats/B161/feedback-analysis.ipynb]: Not sure if that would help.

I did plot the ratings by individual students as a line graph (interpolated so they don't overlap in the graph):

```python
data = marks.T.swaplevel().loc['topic']#.iloc[4:-1]
cols = data.index.copy()
data = data.reset_index(drop=True)
data.index *= 10
for i in range(len(data) * 10 - 9):
    if i % 10:
        data.loc[i] = float('nan')
data = data.interpolate('akima').sort_index()
plot = data.plot(rot=90, legend=False, figsize=(15, 10))
plot.set_xticks([i*10 for i, c in enumerate(cols)])
plot.set_xticklabels(cols);
```

and saw that all the lines are crazy, and are no straight lines at the bottom (which would be students that are permanently unsatisfied):

index

But I guess such students wouldn't fill out the survey at all, so it's pretty pointless to look at the data here.

File: stats/B161/feedback-analysis.ipynb (hunk 1 above)
Comment (encukou) [reply stats/B161/feedback-analysis.ipynb:14]: Added.

**Comment by encukou** [comment]
Thanks for the review!

File: stats/B161/feedback-analysis.ipynb (hunk 2 above)
Comment (hroncok) [reply stats/B161/feedback-analysis.ipynb]: dada art

**Pull request merged**